docs/library/machine.rst

```diff
@@ -99,6 +98,20 @@ Power related functions
    * A lightsleep has full RAM and state retention.  Upon wake execution is resumed
      from the point where the sleep was requested, with all subsystems operational.
 
+.. function:: deepsleep([time_ms])
```

Comment:
I think it's clearer when these are documented together.  They have effectively the same documentation.

When documented separately there's a lot of duplicated text.